test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/Search/RevIncludeSearchTests.cs

```diff
@@ -0,0 +1,228 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.Web;
+using Hl7.Fhir.Model;
+using Microsoft.Health.Fhir.Client;
+using Microsoft.Health.Fhir.Shared.Tests.E2E.Rest.Search;
+using Microsoft.Health.Fhir.Tests.Common.FixtureParameters;
+using Xunit;
+using Task = System.Threading.Tasks.Task;
+
+namespace Microsoft.Health.Fhir.Tests.E2E.Rest.Search
+{
+    [HttpIntegrationFixtureArgumentSets(DataStore.SqlServer, Format.Json)]
+    public class RevIncludeSearchTests : SearchTestsBase<RevIncludeSearchTestFixture>
+    {
+        public RevIncludeSearchTests(RevIncludeSearchTestFixture fixture)
+            : base(fixture)
+        {
+        }
+
+        [Fact]
+        public async Task GivenARevIncludeSearchExpression_WhenSearched_ThenCorrectBundleShouldBeReturned()
```

Comment:
Don'tt you want to test the QueryGenerator to verify that the resulting SQL Expression is as expected?

Reply:
talked f2f. will be addressing in another pr